gap/Parser.gi

```diff
@@ -585,27 +585,30 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
             if not IsBound( tree!.TitlePage.TitleComment ) then
                 tree!.TitlePage.TitleComment := [ ];
             fi;
-            tree!.content := tree!.TitlePage.TitleComment;
+            current_item := tree!.TitlePage.TitleComment;
 
-            Add( tree, "<URL>" );
-            Add( tree, current_command[ 2 ] );
-            Add( tree, "</URL>" );
+            Add( current_item, "<URL>" );
+            Add( current_item, current_command[ 2 ] );
+            Add( current_item, "</URL>" );
```

Comment:
So, I actually wonder about this @URL -- it is listed in the AutoDoc manual in section "2.3 Title page commands" together with all the other title page commands. But clearly it is different from the rest, since you have to implement it differently. And indeed, I just had a look at the GAPDoc manual, and there is no URL element of the title page.

So, section "2.3 Title page commands" should be adapted, documenting @URL differently.

Or perhaps @URL should be abolished -- I don't understand what it is good for, why can't you just write
```
#! @TitleComment <URL>http://bla</URL>
```
or even (if we automatically turned URLs into links)
```
#! @TitleComment http://bla
```
After all, is it really that common that one wants to put an URL into a TitleComment that it justifies having a dedicated AutoDoc command for it?